src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  // const menAges = (century)
+  //   ? people
+  //     .filter(person => Math.ceil(person.died / 100) === century
+  //       && person.sex === 'm')
+  //   : people
+  //     .filter(person => person.sex === 'm');
+
+  const mens = people.filter(person => person.sex === 'm');
+  const menAges = (century)
```

Comment:
Consider renaming the variable `mens` to `men` for better readability and consistency with other variable names.